src/toolchain/web-guide.md

```diff
@@ -0,0 +1,135 @@
+<!--
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+
+# Intro to Web Builds
+
+Web builds are a fair bit more to get started with compared to native builds.
+This will be a complete guide on how to get things compiled.
+Setting up a web server is out-of-scope for this guide, and best found elsewhere.
+
+- Web support with `gdext` is experimental and should be understood as such before proceeding.
```

Comment:
Do the gdext docs have support for warning boxes? I believe that'd be ideal here. Otherwise, we could add something in bold like **WARNING:**.

Also, I think it may be interesting to point out explicitly here that it "may have bugs we aren't aware of".

Reply:
I believe you'll get a compile error if you don't enable the `experimental-wasm` feature flag.  Beyond that I'm not sure.

Reply:
We do have warning boxes, see ` ```admonish ` tags elsewhere.

But only the 2nd bullet point is a warning. The first can also be moved up to the "This will be a complete guide..." section.